code/modules/clothing/suits/armor.dm

```diff
@@ -312,3 +312,162 @@
+		return
+
+	if(owner.get_item_by_slot(ITEM_SLOT_HEAD) && !istype(owner.get_item_by_slot(ITEM_SLOT_HEAD), /obj/item/clothing/head/soft/sec/nemesis))
+		to_chat(owner, "<span class='warning'>WARNING: Detected unrecommended headgear. Please use Nemesis Solutions softcap for optimal effect.</span>")
+
+	if(!do_after(owner, 1 SECONDS, target = src) || !owner.equip_to_slot_if_possible(gloves, ITEM_SLOT_GLOVES))
+		to_chat(owner, "<span class='warning'>ERROR: Unable to unfold the gloves. ABORTING.</span>")
+		return
+	playsound(src, 'sound/mecha/mechmove03.ogg', 50, TRUE)
+
+	if(!do_after(owner, 1 SECONDS, target = src) || !owner.equip_to_slot_if_possible(boots, ITEM_SLOT_FEET))
+		to_chat(owner, "<span class='warning'>ERROR: Unable to unfold the boots. ABORTING.</span>")
+		owner.dropItemToGround(gloves, TRUE)
+		gloves.forceMove(src)
+		return
+	playsound(src, 'sound/mecha/mechmove03.ogg', 50, TRUE)
+
+	if(!do_after(owner, 1 SECONDS, target = src) || !owner.equip_to_slot_if_possible(belt, ITEM_SLOT_BELT))
+		to_chat(owner, "<span class='warning'>ERROR: Unable to unfold the belt. ABORTING.</span>")
+		owner.dropItemToGround(gloves, TRUE)
+		gloves.forceMove(src)
+		owner.dropItemToGround(boots, TRUE)
+		boots.forceMove(src)
+		return
+	playsound(src, 'sound/mecha/mechmove03.ogg', 50, TRUE)
+	active = TRUE
+
+	gloves.update_charge()
+
+	to_chat(owner, "<span class='notice'>Suit successfully activated.</span>")
+
+/obj/item/clothing/suit/armor/vest/nemesis/proc/fold()
+	if(!ishuman(loc))
+		return
+	var/mob/living/carbon/human/owner = loc
+
+	to_chat(owner, "<span class='notice'>Attempting to fold the suit</span>")
+
+	if(!do_after(owner, 5 SECONDS, target = src))
+		to_chat(owner, "<span class='warning'>ERROR: Please retain from moving while the suit is deactivating. ABORTING.</span>")
+		return
+
+	playsound(src, 'sound/mecha/mechmove03.ogg', 50, TRUE)
+
+	active = FALSE
+
+	owner.dropItemToGround(gloves, TRUE)
+	gloves.forceMove(src)
+
+	owner.dropItemToGround(boots, TRUE)
+	boots.forceMove(src)
+
+	owner.dropItemToGround(belt, TRUE)
+	belt.forceMove(src)
+
+/obj/item/clothing/suit/armor/vest/nemesis/dropped(mob/user)
+	. = ..()
+	if(!ishuman(loc))
+		return
+
+	var/mob/living/carbon/human/owner = loc
+
+	playsound(src, 'sound/mecha/mechmove03.ogg', 50, TRUE)
+
+	active = FALSE
+
+	owner.dropItemToGround(gloves, TRUE)
+	gloves.forceMove(src)
+
+	owner.dropItemToGround(boots, TRUE)
+	boots.forceMove(src)
+
+	owner.dropItemToGround(belt, TRUE)
+	belt.forceMove(src)
+
+/obj/item/clothing/suit/armor/vest/nemesis/proc/update_charge(charge = 0)
+	if(charge > 0)
+		icon_state = "nemesis_powered"
+		boots.icon_state = "nemesis_powered"
+	else
+		icon_state = "nemesis"
+		boots.icon_state = "nemesis"
+	update_icon()
+	boots.update_icon()
+	if(isliving(loc))
+		var/mob/living/owner = loc
+		owner.update_appearance()
+
+/obj/item/clothing/suit/armor/vest/nemesis/attack_hand(mob/user, list/modifiers)
```

Comment:
so, uh, do you have to click+drag this into your hand if you want to unequip it?

Reply:
Yes.